### ISSUE TITLE
Seems like you can't download specific episodes with aria2c?

### ISSUE THREAD

**Issue by ShiraazMoollatjie** [vn-ki/anime-downloader#112]
**Describe the bug**
Seems like I can't download a range of episodes with aria2c?

**To reproduce**
```
anime dl 'fullmetal' -xd {aria2} -e 27:35

CHOOSE => 5: Fullmetal Alchemist: Brotherhood        	Ep 64/64 
```

It works when I either:
* Not use the `-xd` option
* Not use the `-e` option

It won't work when both options are used together.

<details>
<summary>LOG</summary>

```

Traceback (most recent call last):
  File "/home/shiraaz/python-environments/anime-dl/bin/anime", line 11, in <module>
    sys.exit(cli())
  File "/home/shiraaz/python-environments/anime-dl/lib/python3.6/site-packages/click/core.py", line 764, in __call__
    return self.main(*args, **kwargs)
  File "/home/shiraaz/python-environments/anime-dl/lib/python3.6/site-packages/click/core.py", line 717, in main
    rv = self.invoke(ctx)
  File "/home/shiraaz/python-environments/anime-dl/lib/python3.6/site-packages/click/core.py", line 1137, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/home/shiraaz/python-environments/anime-dl/lib/python3.6/site-packages/click/core.py", line 956, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/home/shiraaz/python-environments/anime-dl/lib/python3.6/site-packages/click/core.py", line 555, in invoke
    return callback(*args, **kwargs)
  File "/home/shiraaz/python-environments/anime-dl/lib/python3.6/site-packages/click/decorators.py", line 17, in new_func
    return f(get_current_context(), *args, **kwargs)
  File "/home/shiraaz/python-environments/anime-dl/lib/python3.6/site-packages/anime_downloader/cli.py", line 135, in dl
    episode.ep_no, anime.title)
AttributeError: 'list' object has no attribute 'title'

```

**Comment by hazique94** [comment]
It seems have problem with the logging. When I comment out the logging info, it can be downloaded using aria for episodes range but it will not show this dialog in the command. 

"Downloading episode 1 of Fighting Spirit" 

Hope the dev can fixed in next release.

**Comment by moosemedia** [comment]
I have been having the same problem, I didn't think to do it without the episodes defined!  I figured there was an issue with my PC!

**Comment by vn-ki** [comment]
Should be fixed in 3.6.2. Please check.

**Comment by ShiraazMoollatjie** [comment]
@vn-ki it's working now :)
